index.html

```diff
@@ -237,5 +237,22 @@ <h4>Skills</h4>
           </div>  
         </tag>
     </section>
+     <section class="contact-container">
+        <div class="contact-me">
+          <h2 class="contact-heading">Contact me</h2>
+          <p class="contact-text">If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love to help with it.
+           </p>
+        <div class="form">
+          <form action="submit" class="contact-form">
+            <input class="contact-input"  value="Leslie" type="text" >
+            <input class="contact-input" type="email" value="leeegd99@gmail.com">
```

Comment:
- [ ] Please add a `required` attribute to this email input to trigger the validations discussed above. 🙂️
- [ ] Kindly replace `value` attribute with `placeholder` for the reason discussed earlier. 🧐️